openslides_backend/action/generics.py

```diff
@@ -336,9 +335,7 @@ def delete_action_prepare_dataset(self, payload: ActionPayload) -> DataSet:
             relation_fields: List[Tuple[str, BaseRelationField]] = []
             # Gather all delete actions with payload and also all models to be deleted
             delete_actions: List[Tuple[Type[Action], ActionPayload]] = []
-            additional_relation_models: ModelMap = deepcopy(
```

Comment:
Please check this. Do we have to copy the given additional_relation_models here or not? I am not sure. If you are sure it is ok.

Reply:
Yes, we have to copy them, but this was moved to the `execute_other_action` method (where the copying is done via he `**` operator).